package.json

```diff
@@ -142,8 +150,10 @@
       "icon": "assets"
     }
   },
-  "prettier": {
-    "singleQuote": true,
-    "trailingComma": "all"
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged",
```

Comment:
not necessary but should we use `lint-staged` to lint only those files that are staged? https://github.com/okonet/lint-staged

Reply:
Yes. Good idea